src/NUnitFramework/framework/Internal/Randomizer.cs

```diff
@@ -611,6 +611,25 @@ public decimal NextDecimal(decimal min, decimal max)
             return unchecked(raw % range + min);
         }
 
+        #endregion 
+
+        #region Guid
+
+        /// <summary>
+        /// Returns a random <see cref="Guid"/>.
```

Comment:
> just like Guid.NextGuid()
I'm sorry but I don't understand what you mean by that. Do you think it is necessary to talk about `Guid.NewGuid()` here ? It seems that the guids created by `Guid.NewGuid()` always are v4 guids but I don't see that specified anywhere in the documentation, even in the documentation of `CoCreateGuid`.